http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/model/LeonardoModel.scala

```diff
@@ -362,7 +362,8 @@ final case class ClusterTemplateValues private (googleProject: String,
                                                 googleClientId: String,
                                                 welderEnabled: String,
                                                 notebooksDir: String,
-                                                customEnvVarsConfigUri: String) {
+                                                customEnvVarsConfigUri: String,
+                                                memLimit: String) {
```

Comment:
👍

Reply:
I think this field should be `String` by design because this is for templating values… this class is populated with the literal `String`s to be templated.

However I did make `MemorySize` and am using it instead of `Int` in other places.